src/pe_source/cybersixgill.py

```diff
@@ -127,6 +131,39 @@ def get_alerts(self, org_id, sixgill_org_id, pe_org_uid, source_uid):
         except Exception as e:
             logging.error("Failed fetching alert data for %s", org_id)
             logging.error(e)
+            print(traceback.format_exc())
+            return 1
+
+        # Get Alert content
+        try:
+            logging.info("Fetching alert content data for %s.", org_id)
+            # Fetch organization assets
+            org_assets_dict = all_assets_list(sixgill_org_id)
+            print(org_assets_dict)
```

Comment:
Why are you using `print()` rather than `logging`, here and elsewhere in this PR?

Reply:
resolved: [036b1d9](https://github.com/cisagov/pe-reports/pull/193/commits/036b1d9ca88dc6966e5914e5ae658140e7ec568c)